pyogrio/geopandas.py

```diff
@@ -44,7 +46,7 @@ def read_dataframe(
     sql=None,
     sql_dialect=None,
     fid_as_index=False,
-    use_arrow=False,
+    use_arrow=None,
```

Comment:
I would prefer to keep the docstring saying "default False", because that is still the case by _default_ for users because by default you don't set the env variable. But will certainly update the docstring to mention the env variable.